Update otlp-log-viewer API response and README (#14)
dash0hq/take-home-assignments: otlp-log-viewer/README.md

**Endpoint:**
```
GET https://take-home-assignment-otlp-logs-api.vercel.app/api/v2/logs
```

**TypeScript Types:**

> The response conforms to the [OTLP Logs protobuf schema](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto). The `@opentelemetry/otlp-transformer` package (version noted above) is a good starting point for TypeScript types.

Reviewer: ```suggestion
The response conforms to the [OTLP Logs protobuf schema](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto).
You can use it to generate the necessary types
```

The otlp-transformer package no longer exports the types, we should remove it
